webaudio/resources/audit.js

```diff
@@ -2,6 +2,9 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// See https://github.com/web-platform-tests/wpt/issues/12781 for information on
+// the purpose of audit.js, and why testharness.js does not suffice.
+// https://github.com/web-platform-tests/wpt/issues/12781
```

Comment:
Is there a need for a redundant link from LOC 5?

Reply:
Nope, I just accepted Stephen's suggestion which only updated the first two lines. Fixed now.